Library/Formula/quantlib.rb

```diff
@@ -22,6 +22,13 @@ class Quantlib < Formula
 
   option :cxx11
 
+  # fix for quantlib 1.7 linking conflicts with the boost thread library
+  # this patch must be removed when quantlib 1.8 is released, because quantlib maintainers fixed the bug
```

Comment:
Here it is: https://github.com/lballabio/QuantLib/commit/d1909593d9f36c6703966460fb48773792facd7e